src/contents/Prove-Online-Facts.mdx

```diff
@@ -0,0 +1,298 @@
+---
+name: "Prove Online Facts Without Revealing Too Much"
+index: 12
+summary: Learn to prove online facts securely without exposing sensitive information using TLSNotary.
+author: Chiemezie Agbo
+authorIcon: https://x.com/ChiemezieAgbo/photo
```

Comment:
Please use a direct link to your image. This should work:
`https://pbs.twimg.com/profile_images/1697221657373360128/iqDhBtk9_200x200.jpg`